spec/helpers/blacklight_helper_spec.rb

```diff
@@ -5,8 +5,30 @@
 describe BlacklightHelper do
   let(:blacklight_params) { {} }
 
+  class FakeSearchState
```

Comment:
There is probably a way to stub this via FactoryWhatever. Let me know if you know how.